### DISCUSSION TITLE
WSA v2304.40000.4.0 has been released

### DISCUSSION THREAD

**Discussion by Almighty188** [LSPosed/MagiskOnWSALocal#537] [General]
On the Windows side, get it from:
Download https://github.com/MustardChef/WSAPackages/releases/download/WSA_2304.40000.4.0/MicrosoftCorporationII.WindowsSubsystemForAndroid_2304.40000.4.0_neutral_._8wekyb3d8bbwe.Msixbundle
with a big thanks to @MustardChef for providing the file archive.
Rename the downloaded file to wsa-WIF.zip
Delete everything inside the download folder for MagiskOnWSALocal
Place the wsa-WIF.zip file in the MagiskOnWSALocal download folder

In Linux, Build MagiskOnWSALocal as usual and remember to choose the fast channel for WSA

**Comment by williamvyong** [comment]
im not sure whats the problem i cant create erofs image 
ERROR: Failed to make erofs image from /tmp/wsa-build-oUosyOPFuj_/system_root_merged/vendor
Build: an error has occurred, exit

**Reply by icefrog119** [reply]
I'm getting Build: an error has occurred, exit    just like the previous comment above me said

**Reply by Almighty188** [reply]
You need to update the erofs-utils to atleast v1.4.   

See discussion below:
https://github.com/LSPosed/MagiskOnWSALocal/issues/535
